operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakIngressTest.java

```diff
@@ -188,4 +192,108 @@ public void testMainIngressDurability() {
                     assertThat(k8sclient.network().v1().ingresses().inNamespace(namespace).list().getItems().size()).isEqualTo(0);
                 });
     }
+
+    @Test
+    public void testCustomIngressDeletion() {
```

Comment:
I think this test can be significantly simplified. Something like:
1. Deploy custom ingress (without the owner reference but with the same name as if it was created by the Operator leveraging `Constants.KEYCLOAK_INGRESS_SUFFIX`).
2. Deploy Keycloak with ingress disabled.
3. Check if the custom ingress is not deleted.

Reply:
Done!

Thanks.